packages/iris-grid/src/IrisGridModelFactory.js

```diff
@@ -7,6 +7,7 @@ import IrisGridProxyModel from './IrisGridProxyModel';
 class IrisGridModelFactory {
   /**
    * @param {Table|TreeTable} table The Table or TreeTable object to create the model with
+   * @param {boolean} isConsoleTable If the table is a console table
```

Comment:
I had code before your JS plugins PR that was being type checked by CRA. Prior to that change the method had 3 params and only 2 documented, but my rebase removed it and for some reason CRA didn't complain? Anyway I'm removing this comment since the param was removed